src/Annotation/Annotation.php

```diff
@@ -4,6 +4,13 @@
 
 namespace Overblog\GraphQLBundle\Annotation;
 
-interface Annotation
+use Doctrine\Common\Annotations\AnnotationException;
+
+abstract class Annotation
 {
+    protected function cumulatedAttributesException(string $attribute, string $value, string $attributeValue): void
+    {
+        $annotationName = str_replace('Overblog\GraphQLBundle\Annotation\\', '', get_class($this));
```

Comment:
```suggestion
        $annotationName = (new ReflectionClass($this))->getShortName();
```

Reply:
I can't because of Relay Annotations with a different namespace (https://github.com/overblog/GraphQLBundle/blob/master/src/Annotation/Relay/Connection.php)